bootstrap/updater.go

```diff
@@ -143,51 +145,57 @@ func (u *Updater) Load(ctx context.Context) error {
 		if err = u.updateAndNotify(ctx, verified); err != nil {
 			return err
 		}
-		u.logger.With().Info("loaded boostrap file", log.Inline(verified))
+		u.logger.With().Info("loaded bootstrap file", log.Inline(verified))
 		u.addUpdate(verified.Data.Epoch, verified.Persisted[len(verified.Persisted)-suffixLen:])
 	}
 	return nil
 }
 
-func (u *Updater) Start(ctx context.Context) error {
+func (u *Updater) Start() error {
 	if len(u.cfg.DataDir) == 0 {
 		return fmt.Errorf("data dir not set %s", u.cfg.DataDir)
 	}
 	u.once.Do(func() {
 		u.eg.Go(func() error {
+			ctx := log.WithNewSessionID(context.Background())
 			if err := u.Load(ctx); err != nil {
 				return err
 			}
-			wait := time.Duration(0)
```

Comment:
this is intended to make it run immediately the first time on startup. this data is important for other component and should be read asap

Reply:
Updated